packages/components/Checkbox/src/CheckboxTokens.android.ts

```diff
@@ -13,16 +13,16 @@ export const defaultCheckboxTokens: TokenSettings<CheckboxTokens, Theme> = (t: T
     checkmarkSize: globalTokens.font.size100,
     label: {
       color: t.colors.neutralForeground1,
-      padding: globalTokens.spacing.s,
-      paddingHorizontal: globalTokens.spacing.m,
+      padding: globalTokens.size120,
+      paddingHorizontal: globalTokens.size160,
       borderRadius: globalTokens.corner.radius40,
-      spacingLabelAfter: globalTokens.spacing.m,
+      spacingLabelAfter: globalTokens.size160,
       labelIsBefore: {
-        spacingLabelBefore: globalTokens.spacing.s,
+        spacingLabelBefore: globalTokens.size160,
```

Comment:
should be `size120` here as it maps "`s`"?

Reply:
Correct , Thanks 😊  
Updated.